blackbox/suites/fuse/modules/drive9_customer_tencent_perf.py

```
@@ -0,0 +1,884 @@
+        errors: list[str] = []
+        for _ in range(10):
+            start = time.perf_counter()
+            try:
+                with socket.create_connection((host, port), timeout=5):
+                    tcp_values.append((time.perf_counter() - start) * 1000)
+            except OSError as exc:
+                errors.append(str(exc))
+        health: dict[str, Any] = {}
+        start = time.perf_counter()
+        try:
+            with urllib.request.urlopen(server_url.rstrip("/") + "/healthz", timeout=10) as response:
```

Ruff (0.15.17)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)